es-ds-docs/pages/molecules/verification-code.vue

```diff
@@ -0,0 +1,124 @@
+<script setup lang="ts">
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+
+if ($prism) {
+    /* eslint-disable import/no-webpack-loader-syntax, import/no-self-import */
+    const compSource = await import('@energysage/es-ds-components/components/es-verification-code.vue?raw');
+    const docSource = await import('./verification-code.vue?raw');
+    /* eslint-enable import/no-webpack-loader-syntax, import/no-self-import */
+
+    compCode.value = $prism.normalizeCode(compSource.default);
+    docCode.value = $prism.normalizeCode(docSource.default);
+    $prism.highlight();
+}
+
+const propTableRows = [
+    ['v-model', 'String', 'n/a', 'Required. The v-model directive binds the otpinput to a data property.'],
+];
+const code = ref();
+const isSubmitInProgress = ref(false);
+
+const charCount: number = 5;
+
+const isValid = () => {
+    if (code.value && code.value.length === charCount) {
+        return true;
+    }
+    return false;
+};
+
+const randomCode = () => {
+    // Randomly generates an array of numbers 0-9
+    const rand = Array.from({ length: charCount }, () => Math.floor(Math.random() * 10).toString());
+
+    // eslint-disable-next-line no-console
+    console.log('Generated', rand);
```

Comment:
This `console.log` may have existed on the previous docs page, but since these docs pages should be best practice examples of how to code in Nuxt/Vue, I don't think it's appropriate to include a `console.log` here in an example; I'd say it's safe to remove this.